create-from-intercom.mdx

```diff
@@ -5,20 +5,18 @@ description: "Ingest customer conversations and support tickets to Monterey for
 
 ![intercom](/images/inter.png)
 
-## Set up
+**Set up**
 
-You can find the Intercom integration on your [integration settings page](https://app.monterey.ai/w/default/settings/integrations/). Follow the instructions to connect your Intercom account. Anyone who has access to your Monterey workspace can set up Intercom integration.
+You can find Intercom integration on your **[intigration setting page](https://docs.monterey.ai/https://app.monterey.ai/w/cl5b6pdc45011ebfde64e4qkf/settings/integrations/)**. Follow the insturtion to connet your Intercom account. Anyone who has access to your Monterey workspace can set up Intercom integration.
```

Comment:
Anywhere you link to `/w/cl5b6...` it will land the user on a 404 page because its a link to a specific workspace. Have a routing fix that will allow entering `/w/default/...` and that will resolve and redirect the user to a workspace they actually have access to, so be sure to update any links in docs to go to `/w/default/...` else the link will land the user on a 404 page and not the place you intend them to.